notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>You might want to fold this part into the <code>Span</code> section  above. Perhaps show a single example of something from the <code>spanner</code>  package, then refer to the API docs (https://text-extensions-for-pandas.readthedocs.io/en/latest/#module-text_extensions_for_pandas.spanner) for more information.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I came up with an example, I think it flows pretty good but let me know what you think</p>

Reply:
It does flow better to have this after the spans, so I moved it there.